data/starksheet.json

```
@@ -0,0 +1,74 @@
+{
+  "name": "Starksheet",
+  "description": "Like a spreadsheet, powered by the open web: each cell can call any contract using any on-chain data.",
+  "short_description": "The web3 spreadsheet",
+  "tags": [
+    "Onramp",
+    "DeFi",
+    "NFTs",
+    "Social",
+    "DAOs"
+  ], //Remove all that don't apply
+  "contracts": [
```

```suggestion
  "audits": [],
  "tokens": [],
  "contracts": [
```
We need empty arrays for audits and tokens for it to work properly